dlp/snippets/deid_test.py

```diff
@@ -289,3 +289,35 @@ def test_deidentify_with_replace_infotype(capsys):
 
     assert url_to_redact not in out
     assert "My favorite site is [URL]" in out
+
+
+def test_deidentify_with_simple_word_list(capsys):
+    word_list = ["RM-GREEN", "RM-YELLOW", "RM_ORANGE"]
+    deid.deidentify_with_simple_word_list(
+        GCLOUD_PROJECT,
+        "My Room is rm-green",
+        "CUSTOM_ROOM_ID",
+        word_list
+    )
+
+    out, _ = capsys.readouterr()
+
+    assert "rm-green" not in out
```

Comment:
a easier to read test would just assert the entire de-identified string is what is expected, instead of having multiple asserts - that's also more inline how we normally test this

Reply:
Got it, updated in the next commit, Please review.